2020-02-24 20:44:34,6244,ERROR,Error during the environment variables setup:
Traceback (most recent call last):
  File "main.py", line 88, in setup_environment_variables
    with open(conf_file_name) as file:
FileNotFoundError: [Errno 2] No such file or directory: 'conf_devf.txt'
2020-02-24 20:44:37,3308,INFO,Environment variables setup completed (prod env)
2020-02-24 20:44:37,3308,INFO,Closing logger.exe...
2020-02-24 20:44:37,3308,INFO,logger.exe closed
2020-02-24 20:44:37,3308,DEBUG,[]
2020-02-24 20:44:37,3308,INFO,Computer name: DESKTOP-SGGRUD3
2020-02-24 20:44:37,3308,INFO,User name: Hugo
2020-02-24 20:44:37,3308,INFO,Hugo is part of the study
2020-02-24 20:44:37,3308,INFO,Launching logger.exe...
2020-02-24 20:44:37,3308,INFO,logger.exe launched
2020-02-24 20:44:37,3308,DEBUG,['logger.exe                   20968 Console                    4     4ÿ956 Ko\n']
2020-02-24 20:44:37,3308,INFO,Process running...
2020-02-24 20:44:37,3308,INFO,Qualtrics API call to retrieve survey answers...
2020-02-24 20:44:37,3308,INFO,test
2020-02-24 20:44:37,3308,ERROR,surveyId must match ^SV_.*
